frontend/src/metabase/ui/components/inputs/DatePicker/DatePicker.stories.mdx

```diff
@@ -101,6 +102,38 @@ DatesRange.args = {
   <Story name="Dates range">{DatesRange}</Story>
 </Canvas>
 
+### Dates range (SDK theme)
```

Comment:
add a test. It turns out we can't use Storybook interactions with MDX stories, so I didn't add one.